static/app/components/forms/form.tsx

```diff
@@ -235,21 +235,29 @@ function Form({
               )}
 
               <Observer>
-                {() => (
-                  <Button
-                    data-test-id="form-submit"
-                    priority={submitPriority ?? 'primary'}
-                    disabled={
-                      formModel.isError ||
-                      formModel.isSaving ||
-                      submitDisabled ||
-                      (requireChanges ? !formModel.formChanged : false)
-                    }
-                    type="submit"
-                  >
-                    {submitLabel ?? t('Save Changes')}
-                  </Button>
-                )}
+                {() => {
+                  return (
```

Comment:
minor nit: any reason you addd the return? Makes indenting deeper

Reply:
I was probably debugging 🙂 let me remove it